project-management/src/main/java/life/qbic/projectmanagement/domain/model/measurement/NGSSpecificMeasurementMetadata.java

```diff
@@ -0,0 +1,103 @@
+package life.qbic.projectmanagement.domain.model.measurement;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import java.util.Objects;
+import java.util.Optional;
+import life.qbic.projectmanagement.domain.model.sample.SampleId;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+@Embeddable
+public class NGSSpecificMeasurementMetadata {
+
+  @Column(name = "sampleId")
+  private SampleId measuredSample;
+
+  @Column(name = "indexI5")
+  private String indexI5;
+
+  @Column(name = "indexI7")
+  private String indexI7;
+
+  @Column(name = "comment")
+  private String comment;
+
+  protected NGSSpecificMeasurementMetadata() {
+  }
+
+  private NGSSpecificMeasurementMetadata(SampleId measuredSample, String indexI5, String indexI7,
+      String comment) {
+    this.measuredSample = measuredSample;
+    this.indexI5 = indexI5;
+    this.indexI7 = indexI7;
+    this.comment = comment;
+  }
+
+  public static NGSSpecificMeasurementMetadata create(SampleId sampleId, String indexI5,
+      String indexI7, String comment) {
+    Objects.requireNonNull(sampleId);
+    Objects.requireNonNull(indexI5);
+    Objects.requireNonNull(indexI7);
+    Objects.requireNonNull(comment);
```

Comment:
Why do we require non null for the fields?

Reply:
SampleId cannot be null since each measurement has to have at least one associated sampleId.
The other properties are dependent on what the user provides in the spreadsheet, either nothing (ergo an empty string) or content to be stored sample specific in a measurement.